src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseContentAccordionSection.tsx

```diff
@@ -232,13 +235,17 @@ const ReleaseContentAccordionSection = ({
                     Add data block
                   </Button>
                 )}
-                {!showEmbedDashboardForm && (
-                  <Button
-                    variant="secondary"
-                    onClick={toggleEmbedDashboardForm.on}
-                  >
-                    Add embed block
-                  </Button>
+                {user?.permissions.isBauUser && (
```

Comment:
I guess that all our Robot tests are using BAU to embed blocks?

Reply:
Yeah, the only UI test covering embed blocks currently is `admin_and_public_2\bau\publish_release_and_amend.robot` and it's using the BAU user.